src/components/Global/ProposalDropdown.vue

```diff
@@ -1,15 +1,39 @@
 <script setup>
-import { ref, defineEmits } from 'vue'
-import { useUserDataStore } from '../../stores/userData'
+import { ref, defineEmits, computed, defineProps, onMounted } from 'vue'
+import { useProposalStore } from '../../stores/proposalManagement.js'
+
 const emit = defineEmits(['selectionsComplete'])
-const userDataStore = useUserDataStore()
-const proposals = userDataStore.profile.proposals
-const activeProposals = proposals.filter(proposal => proposal.current === true)
+
+const props = defineProps({
+  isItRealTime: {
+    type: Boolean,
+    required: true
+  }
+})
+
+const proposalStore = useProposalStore()
 const selectedProposal = ref()
+const proposals = ref([])
+
+// If the user is requesting a real-time observation, only show proposals with real-time allocations
+// If the user is requesting a normal-time observation, only show proposals with normal time allocations
+const sortProposalDropdownSelection = () => {
+  if (props.isItRealTime) {
+    proposals.value = proposalStore.proposalsWithRealTimeAllocation
+  } else if (!props.isItRealTime) {
+    proposals.value = proposalStore.proposalsWithNormalTimeAllocation
+  }
+}
+
+onMounted(() => {
+  sortProposalDropdownSelection()
```

Comment:
Your not really "sorting" the proposal dropdown selection so much as "setting" it...